internal/commands/pull/command.go

```diff
@@ -168,30 +195,39 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 	}
 	ui.Print(terminal.NewTextLog("Saved app to disk"))
 
-	if cmd.inputs.IncludeDependencies {
+	if cmd.inputs.IncludePackageJSON || cmd.inputs.IncludeNodeModules {
+		logStr := "as a package.json file"
```

Comment:
let's be consistent with our checks (letting `PackageJSON` take precedence)

```go
logStr := "as a node_modules archive"
if cmd.inputs.IncludePackageJSON {
  logStr = "as a package.json file"
}
```

it makes it simpler for us to, less places to ` || IncludeDependencies` on